File: Lista pt3/Exercicio05Arrays.java
public class Exercicio05Arrays {
    public static void main(String[] args) {

        double[] notas = {
                7.5,
                8.0,
                6.5,
                9.0
        };

        double soma = 0;

        for (double nota : notas) {
            soma += nota;
        }

        double media = soma / notas.length;

        System.out.println("Notas:");

        for (double nota : notas) {
            System.out.println(nota);
        }

        System.out.println("Média: " + media);
    }
}

File: Lista pt3/Exercicio05Arrays.js
let notas = [7.5, 8.0, 6.5, 9.0];

let soma = 0;

for (let i = 0; i < notas.length; i++) {
    soma += notas[i];
}

let media = soma / notas.length;

console.log("Notas:");

for (let i = 0; i < notas.length; i++) {
    console.log(notas[i]);
}

console.log("Média: " + media);